templates/zerver/help/browse-and-join-streams.md

```diff
@@ -1,28 +1,29 @@
 # Browse and subscribe to streams
 
 On Zulip, users communicate with each other by sending messages to
```

Comment:
We should change this sentence to 
>  On Zulip, users **primarily** communicate with each other by sending messages to

as users can also communicate using private messages and group pms.